desktop/settings/linux-settings.md

```diff
@@ -0,0 +1,151 @@
+---
+description: Docker Desktop settings
+keywords: Settings, Docker Desktop, Linux
```

Comment:
Perhaps we should add some keywords (include both "settings" and "preferences", to help discoverability); perhaps also add some keywords for the specific sections, as these keywords are used for search auto-complete on the homepage, which helps finding these. e.g.;

```markdown
keywords: settings, preferences, proxy, file sharing, resources, kubernetes, Docker Desktop, Linux
```